tests/invalid_models_tests/test_backend_specific.py

```diff
@@ -5,18 +5,27 @@
 
 from django.core.checks import Error
 from django.db import connection, models
+from django.test import mock
 
 from .base import IsolatedModelsTestCase
 
 
+def dummy_allow_migrate(db, app_label, **hints):
+    # Only allow_migrate on default so we can mock the check_field
```

Comment:
I had a hard time understanding this comment. Maybe something like `# Prevent checks from being run on the 'other' database, which doesn't have its check_field() method mocked in the test.` would be clearer.